### ISSUE TITLE
Incorrect asset path references

### ISSUE THREAD

**Issue by jtwalters** [hollandben/grunt-cache-bust#84]
If you have pages that reference an asset with different paths (e.g. relative paths) then the cached reference will be wrong. 

## Example

Page 1 — /example.html
```html
<script src="./js/script.js"></script>
```
Page 2 — /foo/example.html
```html
<script src="../js/script.js"></script>
```
Page 3 — /foo/bar/example.html
```html
<script src="../../js/script.js"></script>
```

## *Unexpected* Result — Cached result is used in all examples above.
```html
<script src="./js/script.js"></script>
```

**Comment by hollandben** [comment]
This is an issue with the way the task currently replaces the references to files, but my suggestion would be to always use paths from the root as that will always be more reliable, regardless of whether you're using this task or not.

**Comment by hollandben** [comment]
Should have been fixed in #108
